common/tier4_state_rviz_plugin/src/autoware_state_panel.hpp

```diff
@@ -44,7 +44,7 @@ class AutowareStatePanel : public rviz_common::Panel
   explicit AutowareStatePanel(QWidget * parent = nullptr);
   void onInitialize() override;
 
-public Q_SLOTS:
+public Q_SLOTS:  // NOLINT
```

Comment:
Why NOLINT?

Reply:
It occurs `readability-redundant-access-specifiers`.
`public Q_SLOTS` would be needed for Qt. So I cant remove it and add NOLINT.

Reply:
How about adding some supplemental words?

```suggestion
public Q_SLOTS:  // NOLINT for Qt
```